### PR TITLE
DATAENG-987 Add melino json schemas

### PR DESCRIPTION
**Pull request by vchan**
https://radius.atlassian.net/browse/DATAENG-987

### CODE REVIEW

File: src/main/resources/json_schema/stargate/melino/candidates/placeCandidate.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Place Candidate Record from Kinesis",
```
Comment (mrcarmody) [inline src/main/resources/json_schema/stargate/melino/candidates/placeCandidate.json]: I would remove Kinesis from this, doesn't seem to fit.  how about "Place Candidate record from Melino"?

File: src/main/resources/json_schema/stargate/melino/candidates/schemas/place.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Place Schema",
+  "additionalProperties": false,
+  "description": "Place Schema.",
+  "$ref": "#/definitions/place",
+  "definitions": {
+    "place": {
+      "type": "object",
+      "description": "A Place in the Radius Network of Record. Note: the schema [https://github.com/RadiusIntelligence/radius-commons/blob/master/graph-schema/src/main/resources/json_schema/place/place.json] is the ultimate authority for the format of Places. The properties included here are the only the ones relevant to the Melino API, and are a subset of the full object. For the full object, see the schema.",
```
Comment (mrcarmody) [inline src/main/resources/json_schema/stargate/melino/candidates/schemas/place.json]: I would remove the "Note: ...." text.  I don't think that what it describes is true anymore

File: src/main/resources/json_schema/stargate/melino/melinoResponse.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        },
+        "websites": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "addresses": {
+          "type": "array",
+          "description": "fields returned from the USPS API, standardizing the given address.",
```
Comment (mrcarmody) [inline src/main/resources/json_schema/stargate/melino/melinoResponse.json]: might as well update this to indicate that the standardized address is coming from smartystreets

File: src/main/resources/json_schema/stargate/melino/melinoResponse.json [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+              "postalCodeExtra": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
```
Comment (mrcarmody) [inline src/main/resources/json_schema/stargate/melino/melinoResponse.json]: do you know what this is?  i'm guessing this was in the M3 version of this?

Comment (vchan) [reply src/main/resources/json_schema/stargate/melino/melinoResponse.json]: It was. It's a dependency referenced in the `place_match_info` property.

**Pull request merged**